packages/node/rollup.config.js

```diff
@@ -14,6 +16,14 @@ export default [
         sourcemap: true,
         format: "cjs",
         globals: globals
+      },
+      {
+        file: pkg.iife,
+        sourcemap: true,
+        name: "window",
+        format: "iife",
+        extend: true,
+        globals: globals
```

Comment:
Why do you need iife? Won't the Node only be used in modules with existing build processes?

Reply:
If the playground builds it, then sure.

Until then, the poc directly assumed there was a build available and just used it (like this).

cc @joelalejandro